From 21ac942d62f19d5f5a20b0cec4c4753f7d354cae Mon Sep 17 00:00:00 2001
From: Chih-Hsuan Yen <yan12125@gmail.com>
Date: Sun, 15 Dec 2019 09:31:49 +0800
Subject: [PATCH] Fix building with Qt 5.14

---
 lxqtpowermanager.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lxqtpowermanager.cpp b/lxqtpowermanager.cpp
index aef5ca2..b0926f9 100644
--- a/lxqtpowermanager.cpp
+++ b/lxqtpowermanager.cpp
@@ -32,6 +32,7 @@
 #include <QApplication>
 #include <QDesktopWidget>
 #include <QtDebug>
+#include <QFile>
 #include "lxqttranslator.h"
 #include "lxqtglobals.h"
 #include "lxqtsettings.h"
-- 
2.25.0

